README.md

```diff
@@ -101,10 +101,16 @@ sudo dnf install fusesoc
   This is needed to be able to pull down fussoc library components referenced 
   by microwatt. Run
 
+- If you use artix as a reference, like the uart16550 example below, you need to install the corresponding vivado package.
+
+<p align="center">
+<img src="https://github.com/user-attachments/assets/0390507c-cdea-4fce-b2f6-d58830bea176" alt="MicroPython running on Microwatt"/>
```

Comment:
This doesn't seem to be an image of Micropython running on Microwatt, more like Linux booting on Microwatt, so the alt tag is confusing.